File: spray.f90
program spgay
  implicit none
  character(100) :: spgay

  write(*, *) 'SPGAY: '
  read(*, *) spgay

  if (trim(spgay) == '31') then
     write(*, *) 'lol'
  else
     write(*, *) 'aptal'
  end if

end program spgay

File: spray.erl

defmodule Spgay do
  def spgay do
    IO.puts("SPGAY: ")
    spgay = IO.gets("") |> String.trim()

    if spgay == "31" do
      IO.puts("lol")
    else
      IO.puts("aptal")
    end
  end
end

Spgay.spgay()
